ui/ducks/domains.js

```diff
@@ -146,6 +167,7 @@ export function initializeDomainSlice() {
     const network = CHAIN_ID_TO_NETWORK_ID_MAP[chainId];
     const networkName = NETWORK_ID_TO_ETHERS_NETWORK_NAME_MAP[network];
     const ensAddress = networkMap[network];
+    const isSupportedButNotEns = Object.values(CHAIN_IDS).includes(chainId);
```

Comment:
This boolean flag is only ever reached if the `networkIsSupported` flag is not satisfied. This means that at the point that `isSupportedButNotEns` is checked, the currently selected network would be one that we can't call `web3provider.resolveName` on. A snap can however feed resolution into the UI since it's one of the supported networks.

Reply:
I guess my question is when would a snap lookup not be supported? I don't think we need to lock that down to specific chains unless I am missing something?

Reply:
Hmm yeah actually, this is not needed. We can enable domain lookup regardless.